mungegithub/github/github.go

```diff
@@ -143,8 +143,8 @@ type Config struct {
 	Org      string
 	Project  string
 
-	state  string
-	labels []string
+	State  string
+	Labels []string
 
 	Token     string
```

Comment:
We could just make it a private member for now, to avoid logging it. Wouldn't that be work?